### ISSUE TITLE
Unknown sort MInt

### ISSUE THREAD

**Issue by dwightguth** [kframework/k-legacy#873]
```
(error "line 22 column 20: unknown sort 'MInt'")
(error "line 23 column 20: unknown sort 'MInt'")
com.microsoft.z3.Z3Exception: parser error
    at com.microsoft.z3.Native.parseSmtlib2String(Native.java:3014)
    at com.microsoft.z3.Context.ParseSMTLIB2String(Context.java:2330)
    at org.kframework.backend.java.util.Z3Wrapper.checkQuery(Z3Wrapper.java:62)
    at org.kframework.backend.java.symbolic.SymbolicConstraint.checkUnsat(SymbolicConstraint.java:835)
    at org.kframework.backend.java.kil.ConstrainedTerm.unifyImpl(ConstrainedTerm.java:263)
    at org.kframework.backend.java.kil.ConstrainedTerm.unify(ConstrainedTerm.java:213)
    at org.kframework.backend.java.symbolic.SymbolicRewriter.getUnificationResults(SymbolicRewriter.java:385)
    at org.kframework.backend.java.symbolic.SymbolicRewriter.computeRewriteStep(SymbolicRewriter.java:244)
    at org.kframework.backend.java.symbolic.SymbolicRewriter.rewrite(SymbolicRewriter.java:64)
    at org.kframework.backend.java.symbolic.JavaSymbolicKRun.javaKILRun(JavaSymbolicKRun.java:125)
    at org.kframework.backend.java.symbolic.JavaSymbolicKRun.internalRun(JavaSymbolicKRun.java:88)
    at org.kframework.backend.java.symbolic.JavaSymbolicKRun.step(JavaSymbolicKRun.java:326)
    at org.kframework.krun.KRunFrontEnd.normalExecution(KRunFrontEnd.java:135)
    at org.kframework.krun.KRunFrontEnd.run(KRunFrontEnd.java:491)
    at org.kframework.main.FrontEnd.main(FrontEnd.java:40)
    at org.kframework.main.Main.main(Main.java:86)
```

I think the smtlib translation for machine integers is messed up.

**Comment by dwightguth** [comment]
Never mind, I will fix this one since I need to work on it for the deliverable for the integer stuff.

**Comment by andreistefanescu** [comment]
You need to change the SMTLib translation to change sort names. We should probably have an attribute on the sort, and do the renaming in translation pass.

**Comment by dwightguth** [comment]
Fixed #879
